src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionItemResolver.cs

```diff
@@ -39,7 +39,30 @@ internal class RazorCompletionItemResolver : CompletionItemResolver
             return null;
         }
 
-        var associatedRazorCompletion = razorCompletionResolveContext.CompletionItems.FirstOrDefault(completion => string.Equals(completion.DisplayText, completionItem.Label, StringComparison.Ordinal));
+        var associatedRazorCompletion = razorCompletionResolveContext.CompletionItems.FirstOrDefault(completion =>
+        {
+            if (!string.Equals(completion.DisplayText, completionItem.Label, StringComparison.Ordinal))
+            {
+                return false;
+            }
```

Comment:
nit: for an ordinal string comparison, `completion.DisplayText == completionItem.Label` is fine.

Reply:
Makes sense, I just left in place what the original code did there - cleaned it up as suggested now. :)